ton_client/client/src/api/dispatch.rs

```diff
@@ -398,11 +311,14 @@ impl DispatchTable {
     }
 
     pub fn sync_dispatch(
-        &self, context: Arc<ClientContext>, name: String, params_json: String
-    ) -> JsonResponse {
+        &self,
+        context: Arc<ClientContext>,
+        name: String,
```

Comment:
`async_dispatch` receives function name as `function` parameter. Use one naming for both functions

Reply:
renamed